index.js

```diff
@@ -118,6 +140,12 @@ function walker(node, parameters, outerFont) {
           value: node.attribs.color,
           range: [offset, length]
         });
+      } else if (customMatcher !== null) {
```

Comment:
Cool, I'll shift the logic round and pass the customMatcher through as a parameter